docs/reference/operators/ContribOperators.md

```diff
@@ -5,11 +5,23 @@ grand_parent: Reference
 nav_order: 3
 ---
 
+# Contrib ops
+
+## Contents
+{: .no_toc }
+
+* TOC placeholder
+{:toc}
+
+The [contrib ops domain](https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/contrib_ops) contains ops that are built in to the runtime by default. Only selective operators are added as contrib ops to avoid increasing the binary size of the core runtime package. When possible, [custom operators](./add-custom-op.md) should be used.
```

Comment:
```suggestion
The [contrib ops domain](https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/contrib_ops) contains ops that are built in to the runtime by default. Only selected operators are added as contrib ops to avoid increasing the binary size of the core runtime package. When possible, [custom operators](./add-custom-op.md) should be used.
```

Reply:
Updated